packages/ckeditor5-table/tests/manual/tablemocking.html

```diff
@@ -31,17 +31,18 @@
 	<label for="model-data">Table data as expected by <a href="https://github.com/ckeditor/ckeditor5-table/blob/1004f9106110be9de125825afd491a1618b71271/tests/_utils/utils.js#L48">modelTable</a> helper function:</label>
 	<textarea id="model-data">
 [
-	[ '00', '01', '02', '03', '04' ],
-	[ '10', '11', '12', '13', '14' ],
-	[ '20', '21', '22', '23', '24' ],
-	[ '30', '31', '32', '33', '34' ],
-	[ '40', '41', '42', '43', '44' ]
+	[ '00', { contents: '01', colspan: 2, rowspan: 2 }, { contents: '03', rowspan: 2 }, '04', '05' ],
```

Comment:
I would leave simple table as a base here. A simple table is better to start changing its structure. Providing complex table looks like one-case scenario only.